src/app/shared/utils/terms.ts

```diff
@@ -0,0 +1,59 @@
+import dayjs from 'dayjs';
+
+/**
+ * Fetches the readable text version of a term.
+ *
+ * @param {string} term i.e. '202101'
+ * @return {string} 'Spring 2021'
+ */
+export function getReadableTerm(term: string): string {
+  const yearRegex = /\d{4}/g;
+  const monthRegex = /(\d{4})(\d{2})/;
+  const year = yearRegex.exec(term)![0];
+  let month = monthRegex.exec(term)![2];
```

Comment:
I think you can just use the length to get your required values and just note in the documentation you don't support anything otherwise. It's not like the current implementation fails gracefully. I was thinking just using `slice` to grab the values of year/month?

Reply:
fair enough 👍